policyengine_us/parameters/gov/states/nj/tax/income/credits/nj_child_tax_credit/ineligible_age.yaml

```
@@ -0,0 +1,10 @@
+description: New Jersey limits the young child tax credit to filers with children below this age.
```

In case you adapted this from the California YCTC, it differs in name and logic (per child rather than any child)
```suggestion
description: New Jersey provides a child tax credit to filers for each dependent below this age.
```